mainnet/Asset/.json

```diff
@@ -0,0 +1,19 @@
+      "denom": "uccd-core1hnwd78wappjmgygyzr9mwspm7uzytrkcp9pfvv",
```

Comment:
you should add these changes to `mainnet/asset.json` and not create a new folder and file

Reply:
@CoreumCommunityDAO